### PR TITLE
FIX: input csv header can manipulate filenames section [GEAR-344]

### PR DESCRIPTION
**Pull request by NPann**

### CODE REVIEW

File: deid_export/deid_template.py [hunk 1]
```diff
@@ -39,8 +39,13 @@ def find_profile_element(d, target):
     else:
         if tps[0] == 'fields':
             return d['fields'], '.'.join(tps[1:]), True
+        if tps[0] == 'groups':
+            return d['groups'], '.'.join(tps[1:]), True
         else:
-            return find_profile_element(d[tps[0]], '.'.join(tps[1:]))
+            try:
```
Comment (kalebdfischer) [inline deid_export/deid_template.py]: Generally not a good practice to expect exceptions. Should be refactored to specifically handle the int case.

Comment (NPann) [reply deid_export/deid_template.py]: Sometimes easier to ask for forgiveness than permission ;) 
Refactor in 978c084

**Comment by NPann** [comment]
Blasting fast review. Readme updated with an example in 34ec1a1

File: deid_export/deid_template.py (hunk 1 above)
Comment (kalebdfischer) [reply deid_export/deid_template.py]: beautiful

**Pull request merged**